.gitignore

```diff
@@ -38,6 +38,7 @@ __diff_output__
 # Cypress screenshots
 **/cypress/screenshots
 **/cypress/videos
+**/cypress/snapshots/All Specs
```

Comment:
Do we need to escape the space character in the path? Like `".../All\ Spacs"`

Reply:
I believe this works, at least it works on my mac.

https://git-scm.com/docs/gitignore

gitignores are line-based and it looks like only trailing spaces need to be escaped.

Reply:
I am more worried about Windows. I am going to verify it. But everything else looks good!

Reply:
I just verified that it works for windows as well.